### ISSUE TITLE
assertion failure: HSlider sets orientation

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/normal-modes#16]
Related to #2 (code review). With ?ea and after bypassing #14: 

```
assert.js?bust=1581129458624:22 Uncaught Error: Assertion failed: HSlider sets orientation
    at window.assertions.assertFunction (assert.js?bust=1581129458624:22)
    at new HSlider (HSlider.js?bust=1581129458782:27)
    at new NumberControl (NumberControl.js?bust=1581129458782:240)
    at new AmpPhaseAccordionBox (AmpPhaseAccordionBox.js?bust=1581129458782:139)
    at new OneDimensionScreenView (OneDimensionScreenView.js?bust=1581129458782:87)
    at OneDimensionScreen.createView (OneDimensionScreen.js?bust=1581129458782:36)
    at OneDimensionScreen.initializeView (Screen.js?bust=1581129458782:266)
    at Array.<anonymous> (Sim.js?bust=1581129458782:866)
    at Sim.js?bust=1581129458782:874
```

**Comment by pixelzoom** [comment]
This is problematic.  The sim is using `NumberControl` with `sliderOptions: {  orientation: 'vertical', ... }`. and that is not currently supported by `NumberControl`.

**Comment by pixelzoom** [comment]
This was fixed as a side-effect of #14, and is ready for review.   It turned out that  NumberControl was not needed here, just a VSlider.

The lesson to be learned here is when to use NumberControl and when to use Slider.

**Comment by tmildemberger** [comment]
We'll pay more attention in the future to decide if something needs to be a NumberControl or just a Slider. Closing.
